### PR TITLE
Format all Kotlin code with ktfmt.

### PR DESCRIPTION
**Pull request by SanjayVas**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/21)

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -26,10 +26,10 @@ jobs:
         architecture: x64
 
     - name: Set up linters
-      uses: world-federation-of-advertisers/actions/setup-linters@v1.3.0
+      uses: world-federation-of-advertisers/actions/setup-linters@v1.4.0
```
Comment (oliver-amzn) [inline .github/workflows/lint.yml:29]: Why are these broken into two? 

(I know, it's not really part of the PR but I'm interested what the reasoning is)

**Review by oliver-amzn — APPROVED** [review]
I *think* this looks ok.

fyi, for future PRs like this, it's easier to review if you break this into two commits, one for your manual changes (update to GH Actions version, etc) and one for the ktfmt changes.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (SanjayVas) [reply .github/workflows/lint.yml:29]: > fyi, for future PRs like this, it's easier to review if you break this into two commits, one for your manual changes (update to GH Actions version, etc) and one for the ktfmt changes.

You want the enforcement to be lock-step with the one-time global formatting. This way you can say anything after this PR is merged will be formatted.

Comment (SanjayVas) [reply .github/workflows/lint.yml:29]: > Why are these broken into two?

They're separate actions within the same repo that have separate responsibilities and therefore separate types. `setup-linters` is a [JavaScript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) that installs the linters. `lint` is a [composite run steps action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action) that runs a script which lints files changed by a PR.

File: src/main/kotlin/org/wfanet/measurement/common/Flow.kt [hunk 2]
```diff
@@ -132,16 +133,16 @@ private class SingleConsumedFlowItem<T>(singleItem: T) : ConsumedFlowItem<T>() {
...
+ * Consumes the first item of the [Flow], producing a [Flow] for the remaining items.
  *
- * Note that this starts a new coroutine in a separate [CoroutineScope] to
- * produce the returned [Flow] items using a [Channel]. As a result, the
- * returned [Flow] is hot.
+ * Note that this starts a new coroutine in a separate [CoroutineScope] to produce the returned
+ * [Flow] items using a [Channel]. As a result, the returned [Flow] is hot.
  *
  * @return a [ConsumedFlowItem] containing the first item and the [Flow] of
+ * ```
```
Comment (efoxepstein) [inline src/main/kotlin/org/wfanet/measurement/common/Flow.kt:142]: This looks wrong here. I think it's assuming this is code when it's supposed to be a continuation of the `@return`.

File: src/main/kotlin/org/wfanet/measurement/common/RenewedFlow.kt [hunk 3]
```diff
@@ -53,5 +51,4 @@ fun <T> renewedFlow(
   reconnect: Duration,
   reconnectDelay: Duration,
   block: suspend () -> Flow<T>
-): Flow<T> =
-  renewedFlow(reconnect.toMillis(), reconnectDelay.toMillis(), block)
+): Flow<T> = renewedFlow(reconnect.toMillis(), reconnectDelay.toMillis(), block)
```
Comment (efoxepstein) [inline src/main/kotlin/org/wfanet/measurement/common/RenewedFlow.kt:54]: I find this formatting a bit confusing when the parameters are split up, should we just convert this to a block body with `return renewedFlow(...)`?

File: src/main/kotlin/org/wfanet/measurement/common/crypto/SecurityProvider.kt [hunk 4]
```diff
@@ -40,15 +39,17 @@ private val certFactory = CertificateFactory.getInstance(CERTIFICATE_TYPE, jcePr
 fun readCertificate(pemFile: File): X509Certificate {
   return pemFile.inputStream().use { fileInputStream ->
     certFactory.generateCertificate(fileInputStream)
-  } as X509Certificate
+  } as
+    X509Certificate
```
Comment (efoxepstein) [inline src/main/kotlin/org/wfanet/measurement/common/crypto/SecurityProvider.kt:43]: This seems like bad formatting -- is the `as X509Certificate` actually necessary?

File: src/main/kotlin/org/wfanet/measurement/common/identity/DuchyIds.kt [hunk 5]
```diff
@@ -40,11 +38,12 @@ object DuchyIds {
 /**
  * Flag for setting global [DuchyIds.ALL].
  *
- * Usage:
- *   fun run(@CommandLine.Mixin duchyIdFlags: DuchyIdFlags, ...) {
+ * Usage: fun run(@CommandLine.Mixin duchyIdFlags: DuchyIdFlags, ...) {
+ * ```
```
Comment (efoxepstein) [inline src/main/kotlin/org/wfanet/measurement/common/identity/DuchyIds.kt:42]: Weirdness here too

File: src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt [hunk 6]
```diff
@@ -23,19 +23,20 @@ import org.junit.runners.model.Statement
...
+ * [org.junit.rules.ExternalResource.before] throws an exception.
  */
 open class CloseableResource<T : AutoCloseable>(private val createResource: () -> T) : TestRule {
   protected lateinit var resource: T
     private set
 
-  override fun apply(base: Statement, description: Description) = object : Statement() {
-    override fun evaluate() {
-      check(!::resource.isInitialized)
+  override fun apply(base: Statement, description: Description) =
```
Comment (efoxepstein) [inline src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt:33]: Convert to block body?

File: src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt [hunk 7]
```diff
@@ -87,20 +84,26 @@ private fun ConfigMapEntry.toDuchyPublicKeysEntry(): DuchyPublicKeys.Entry {
...
-      publicKeyMap = elGamalElementsMap.mapValues {
-        require(it.value.size() == ELEMENT_SIZE) {
-          "Expected $ELEMENT_SIZE bytes for element. Got ${it.value.size()}."
-        }
-        ElGamalPublicKey.newBuilder().apply {
-          generator = elGamalGenerator
-          element = it.value
-        }.build()
-      },
+      publicKeyMap =
```
Comment (efoxepstein) [inline src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt:87]: This formatting is super confusing, should we pull our arguments to variables?

**Review by efoxepstein — APPROVED** [review]
I ran ktfmt locally and then diffed it with `origin/sanjayvas-ktfmt`. This looks fine to me -- mostly seems like fixing up bugs in the formatter.

```
diff --git a/.editorconfig b/.editorconfig
index 3f75052e..7caf18d4 100644
--- a/.editorconfig
+++ b/.editorconfig
@@ -7,3 +7,9 @@ kotlin_imports_layout=ascii
-# Comma-separated list of rules to disable.
-# indent, import-ordering, and parameter-list-wrapping rules disabled to avoid conflicts with ktfmt.
-disabled_rules=indent,import-ordering,parameter-list-wrapping
+# Argument wrapping currently disabled due to
+#   https://github.com/pinterest/ktlint/issues/926
+disabled_rules=experimental:argument-list-wrapping
+
+[*.kts]
+# Comma-separated list of rules to disable
+# AnnotationRule currently disabled due to
+#   https://github.com/pinterest/ktlint/issues/745
+disabled_rules=experimental:annotation,experimental:argument-list-wrapping
diff --git a/.github/workflows/lint.yml b/.github/workflows/lint.yml
index 2104d6dc..a3db948d 100644
--- a/.github/workflows/lint.yml
+++ b/.github/workflows/lint.yml
@@ -29 +29 @@ jobs:
-      uses: world-federation-of-advertisers/actions/setup-linters@v1.4.0
+      uses: world-federation-of-advertisers/actions/setup-linters@v1.3.0
@@ -32 +32 @@ jobs:
-      uses: world-federation-of-advertisers/actions/lint@v1.4.0
+      uses: world-federation-of-advertisers/actions/lint@v1.3.0
diff --git a/docs/code-style.md b/docs/code-style.md
index 289d5b9a..864609d0 100644
--- a/docs/code-style.md
+++ b/docs/code-style.md
@@ -11 +11,6 @@ The one exception is that we use two spaces instead of four for indentation.
-In addition to the style guide, please observe these guidelines:
+Please keep Kotlin code `ktlint`-formatted. Note that our `.editorconfig` file
+defines just a couple of overrides. This will produce code compliant with the
+style guide.
+
+In addition to the automatic linting and formatting, please observe these
+guidelines:
@@ -53,8 +58 @@ your code prior to submitting.
-Kotlin formatting is done by
-[`ktfmt`](https://github.com/facebookincubator/ktfmt) with the `--google-style`
-option. Please keep code `ktlint`-formatted.
-
-[`ktlint`](https://ktlint.github.io/) is the Kotlin linter, which also has the
-ability to automatically apply some fixes. Our `.editorconfig` file has the
-appropriate settings to ensure that `ktlint` produces code that is compliant
-with the style guide and doesn't conflict with `ktfmt`.
+Kotlin formatting is done by [`ktlint`](https://ktlint.github.io/).
@@ -62,3 +60,5 @@ with the style guide and doesn't conflict with `ktfmt`.
-There is a
-[`ktfmt` IntelliJ plugin](https://plugins.jetbrains.com/plugin/14912-ktfmt) that
-you can install to replace the `Reformat Code` action in IntelliJ IDEs.
+You can set up `ktlint`-compatible formatting in IntelliJ by following
+[these instructions](https://github.com/pinterest/ktlint/blob/master/README.md#-with-intellij-idea).
+Once this is set up, entire directories can be formatted at once by
+right-clicking on the directory in the Project view and selecting "Reformat
+Code".
@@ -95,2 +95,2 @@ Here is how to reference people or issues in different systems:
-*   Issue in a GitHub repository
-    *   `TODO(<org>/<repo>#<number>)`
+*   Issue in this GitHub project
+    *   `TODO(gh/<issue ID>)`
diff --git a/src/main/kotlin/org/wfanet/measurement/common/CommandLines.kt b/src/main/kotlin/org/wfanet/measurement/common/CommandLines.kt
index cf573ba4..f13ff543 100644
--- a/src/main/kotlin/org/wfanet/measurement/common/CommandLines.kt
+++ b/src/main/kotlin/org/wfanet/measurement/common/CommandLines.kt
@@ -26,2 +26,6 @@ import picocli.CommandLine
- * @param command a function annotated with [@Command][CommandLine.Command] to execute
- * @param args command-line arguments
+ * @param command A function annotated with [@Command][CommandLine.Command]
+ * ```
+ *     to execute.
+ * @param args
+ * ```
+ * Command-line arguments.
@@ -36,2 +40,6 @@ fun commandLineMain(command: KFunction<*>, args: Array<String>) {
- * @param command a [Runnable] annotated with [@Command][CommandLine.Command] to execute
- * @param args command-line arguments
+ * @param command A [Runnable] annotated with [@Command][CommandLine.Command] to
+ * ```
+ *     execute.
+ * @param args
+ * ```
+ * Command-line arguments.
diff --git a/src/main/kotlin/org/wfanet/measurement/common/CountDownLatch.kt b/src/main/kotlin/org/wfanet/measurement/common/CountDownLatch.kt
index fd61e918..dec85642 100644
--- a/src/main/kotlin/org/wfanet/measurement/common/CountDownLatch.kt
+++ b/src/main/kotlin/org/wfanet/measurement/common/CountDownLatch.kt
@@ -23 +23,4 @@ import kotlinx.coroutines.Job
- * @param count the number of times [countDown] can be called before a call to [await] would resume
+ * @param count the number of times [countDown] can be called before a call to
+ * ```
+ *     [await] would resume.
+ * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/common/Flow.kt b/src/main/kotlin/org/wfanet/measurement/common/Flow.kt
index 513d9cde..8b11a053 100644
--- a/src/main/kotlin/org/wfanet/measurement/common/Flow.kt
+++ b/src/main/kotlin/org/wfanet/measurement/common/Flow.kt
@@ -39,3 +39 @@ import kotlinx.coroutines.flow.transform
- * ```
- *   Pair(1, "a-1"), Pair(1, "b-1"), Pair(2, "a-2"), Pair(2, "b-2"), Pair(3, "a-3"), Pair(3, "b-3")
- * ```
+ * Pair(1, "a-1"), Pair(1, "b-1"), Pair(2, "a-2"), Pair(2, "b-2"), Pair(3, "a-3"), Pair(3, "b-3")
@@ -66,2 +64,5 @@ fun <T, R> Flow<T>.pairAll(block: suspend (T) -> Flow<R>): Flow<Pair<T, R>> = tr
- * predicate
- * @param onEachBlock block of code to execute for each item in the flow.
+ * ```
+ *    predicate
+ * @param onEachBlock
+ * ```
+ * block of code to execute for each item in the flow.
@@ -96,2 +97,5 @@ fun <T> Flow<T>.withRetriesOnEach(
- * [CoroutineContext][kotlin.coroutines.CoroutineContext] that the flow is collected in.
- * @param concurrency number of Deferred that can be awaiting at once
+ * ```
+ *     [CoroutineContext][kotlin.coroutines.CoroutineContext] that the flow is collected in.
+ * @param concurrency
+ * ```
+ * number of Deferred that can be awaiting at once
diff --git a/src/main/kotlin/org/wfanet/measurement/common/Runfiles.kt b/src/main/kotlin/org/wfanet/measurement/common/Runfiles.kt
index 33e462d6..094c1b12 100644
--- a/src/main/kotlin/org/wfanet/measurement/common/Runfiles.kt
+++ b/src/main/kotlin/org/wfanet/measurement/common/Runfiles.kt
@@ -39,2 +39,4 @@ fun getRuntimePath(runfilesRelativePath: Path): Path? {
- * @param directoryPath normalized path of the directory the library is in, relative to the Bazel
- * runfiles root
+ * @param directoryPath normalized path of the directory the library is in,
+ * ```
+ *     relative to the Bazel runfiles root
+ * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/common/grpc/Channel.kt b/src/main/kotlin/org/wfanet/measurement/common/grpc/Channel.kt
index 73ef1593..088b326d 100644
--- a/src/main/kotlin/org/wfanet/measurement/common/grpc/Channel.kt
+++ b/src/main/kotlin/org/wfanet/measurement/common/grpc/Channel.kt
@@ -37,2 +37,4 @@ fun buildChannel(target: String): ManagedChannel {
- * @param shutdownTimeout duration of time to allow a channel to finish processing on server
- * shutdown
+ * @param shutdownTimeout duration of time to allow a channel to finish
+ * ```
+ *     processing on server shutdown
+ * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/MillBase.kt b/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/MillBase.kt
index 2fe1e7b2..8effd970 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/MillBase.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/MillBase.kt
@@ -77,3 +77,6 @@ import org.wfanet.measurement.system.v1alpha.MetricRequisitionKey
- * @param dataClients clients that have access to local computation storage, i.e., spanner table and
- * blob store.
- * @param globalComputationsClient client of the kingdom's GlobalComputationsService.
+ * @param dataClients clients that have access to local computation storage, i.e., spanner
+ * ```
+ *    table and blob store.
+ * @param globalComputationsClient
+ * ```
+ * client of the kingdom's GlobalComputationsService.
@@ -83,2 +86,5 @@ import org.wfanet.measurement.system.v1alpha.MetricRequisitionKey
- * computation table.
- * @param computationType The [ComputationType] this mill is working on.
+ * ```
+ *    computation table.
+ * @param computationType
+ * ```
+ * The [ComputationType] this mill is working on.
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/liquidlegionsv2/LiquidLegionsV2Mill.kt b/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/liquidlegionsv2/LiquidLegionsV2Mill.kt
index 5bcde231..4b4fffff 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/liquidlegionsv2/LiquidLegionsV2Mill.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/liquidlegionsv2/LiquidLegionsV2Mill.kt
@@ -80,3 +80,6 @@ import org.wfanet.measurement.system.v1alpha.LiquidLegionsV2
- * @param dataClients clients that have access to local computation storage, i.e., spanner table and
- * blob store.
- * @param metricValuesClient client of the own duchy's MetricValuesService.
+ * @param dataClients clients that have access to local computation storage, i.e., spanner
+ * ```
+ *    table and blob store.
+ * @param metricValuesClient
+ * ```
+ * client of the own duchy's MetricValuesService.
@@ -86,2 +89,5 @@ import org.wfanet.measurement.system.v1alpha.LiquidLegionsV2
- * computation table.
- * @param requestChunkSizeBytes The size of data chunk when sending result to other duchies.
+ * ```
+ *    computation table.
+ * @param requestChunkSizeBytes
+ * ```
+ * The size of data chunk when sending result to other duchies.
@@ -90,2 +96,5 @@ import org.wfanet.measurement.system.v1alpha.LiquidLegionsV2
- * computationControlClients, used for passing computation to other duchies.
- * @param cryptoKeySet The set of crypto keys used in the computation.
+ * ```
+ *    computationControlClients, used for passing computation to other duchies.
+ * @param cryptoKeySet
+ * ```
+ * The set of crypto keys used in the computation.
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/daemon/utils/DuchyOrder.kt b/src/main/kotlin/org/wfanet/measurement/duchy/daemon/utils/DuchyOrder.kt
index 42b56287..1cd64db0 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/daemon/utils/DuchyOrder.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/daemon/utils/DuchyOrder.kt
@@ -43 +43,3 @@ fun getDuchyOrderByPublicKeysAndComputationId(
- * the first duchy will be returned.
+ * ```
+ *    the first duchy will be returned.
+ * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationDataClients.kt b/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationDataClients.kt
index c6b4345e..50980089 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationDataClients.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationDataClients.kt
@@ -71,2 +71,4 @@ private constructor(
-   * @return the resulting [ComputationToken] after updating blob reference, or [computationToken]
-   * if no blob was written
+   * @return the resulting [ComputationToken] after updating blob reference, or
+   * ```
+   *     [computationToken] if no blob was written
+   * ```
@@ -97,4 +99,9 @@ private constructor(
-   * @param writeContent function which writes bound blob content, to be called with
-   * [computationToken]
-   * @return resulting [ComputationToken] from write, or [computationToken] if no write was
-   * performed
+   * @param writeContent function which writes bound blob content, to be called
+   * ```
+   *     with [computationToken]
+   * @return
+   * ```
+   * resulting [ComputationToken] from write, or [computationToken] if
+   * ```
+   *     no write was performed
+   * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationsDatabase.kt b/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationsDatabase.kt
index 2b905dd8..fdeb3de3 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationsDatabase.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/ComputationsDatabase.kt
@@ -94,7 +94,16 @@ interface ComputationsDatabaseTransactor<ProtocolT, StageT, StageDetailsT, Compu
-   * @param inputBlobPaths References to BLOBs that are inputs to this computation stage, all inputs
-   * should be written on transition and should not change.
-   * @param passThroughBlobPaths References to BLOBs that are outputs of this computation stage, but
-   * were written before the start of the computation stage.
-   * @param outputBlobs Number of BLOBs this computation outputs. These are created as part of the
-   * computation so they do not have a reference to the real storage location.
-   * @param afterTransition The work to be do with the computation after a successful transition.
+   * @param inputBlobPaths References to BLOBs that are inputs to this computation stage, all
+   * ```
+   *    inputs should be written on transition and should not change.
+   * @param passThroughBlobPaths
+   * ```
+   * References to BLOBs that are outputs of this computation stage,
+   * ```
+   *    but were written before the start of the computation stage.
+   * @param outputBlobs
+   * ```
+   * Number of BLOBs this computation outputs. These are created as
+   * ```
+   *    part of the computation so they do not have a reference to the real storage location.
+   * @param afterTransition
+   * ```
+   * The work to be do with the computation after a successful transition.
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/db/metricvalue/MetricValueDatabase.kt b/src/main/kotlin/org/wfanet/measurement/duchy/db/metricvalue/MetricValueDatabase.kt
index 68147485..00eab226 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/db/metricvalue/MetricValueDatabase.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/db/metricvalue/MetricValueDatabase.kt
@@ -25,3 +25,9 @@ interface MetricValueDatabase {
-   * @param metricValue the [MetricValue] to insert. The [externalId][MetricValue.getExternalId]
-   * field will be ignored.
-   * @return the inserted [MetricValue] with a new [externalId][MetricValue.getExternalId] assigned.
+   * @param metricValue the [MetricValue] to insert. The
+   * ```
+   *     [externalId][MetricValue.getExternalId] field will be ignored.
+   * @return
+   * ```
+   * the inserted [MetricValue] with a new
+   * ```
+   *     [externalId][MetricValue.getExternalId] assigned.
+   * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/deploy/gcloud/spanner/computation/GcpSpannerComputationsDatabaseTransactor.kt b/src/main/kotlin/org/wfanet/measurement/duchy/deploy/gcloud/spanner/computation/GcpSpannerComputationsDatabaseTransactor.kt
index d9091636..7980c06d 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/deploy/gcloud/spanner/computation/GcpSpannerComputationsDatabaseTransactor.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/deploy/gcloud/spanner/computation/GcpSpannerComputationsDatabaseTransactor.kt
@@ -604 +604,4 @@ class GcpSpannerComputationsDatabaseTransactor<
-   * @throws IllegalStateException if the token is not for the most recent update
+   * @throws IllegalStateException if the token is not for the most recent
+   * ```
+   *     update
+   * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/duchy/storage/ComputationStore.kt b/src/main/kotlin/org/wfanet/measurement/duchy/storage/ComputationStore.kt
index d744a48f..7b3278f4 100644
--- a/src/main/kotlin/org/wfanet/measurement/duchy/storage/ComputationStore.kt
+++ b/src/main/kotlin/org/wfanet/measurement/duchy/storage/ComputationStore.kt
@@ -31,2 +31,4 @@ private typealias ComputationBlobKeyGenerator = ComputationToken.() -> String
- * @param generateBlobKey a function to generate a unique blob key for a given [token]
- * [ComputationToken].
+ * @param generateBlobKey a function to generate a unique blob key for a given
+ * ```
+ *     [token][ComputationToken].
+ * ```
diff --git a/src/main/kotlin/org/wfanet/measurement/gcloud/spanner/testing/SpannerEmulator.kt b/src/main/kotlin/org/wfanet/measurement/gcloud/spanner/testing/SpannerEmulator.kt
index 21bca8a5..49e60129 100644
--- a/src/main/kotlin/org/wfanet/measurement/gcloud/spanner/testing/SpannerEmulator.kt
+++ b/src/main/kotlin/org/wfanet/measurement/gcloud/spanner/testing/SpannerEmulator.kt
@@ -34 +34,4 @@ private const val INVALID_HOST_MESSAGE =
- * @param port TCP port that the emulator should listen on, or 0 to allocate a port automatically
+ * @param port TCP port that the emulator should listen on, or 0 to allocate a
+ * ```
+ *     port automatically
+ * ```
@@ -59,3 +62,8 @@ class SpannerEmulator(private val port: Int = 0) : AutoCloseable {
-   * [kotlinx.coroutines.TimeoutCancellationException].
-   * @return the emulator host, which can be passed to
-   * [com.google.cloud.spanner.SpannerOptions.Builder.setEmulatorHost].
+   * ```
+   *     [kotlinx.coroutines.TimeoutCancellationException].
+   * @return
+   * ```
+   * the emulator host, which can be passed to
+   * ```
+   *    [com.google.cloud.spanner.SpannerOptions.Builder.setEmulatorHost].
+   * ```
diff --git a/src/test/kotlin/org/wfanet/measurement/duchy/daemon/herald/HeraldTest.kt b/src/test/kotlin/org/wfanet/measurement/duchy/daemon/herald/HeraldTest.kt
index d8979946..7d4380f2 100644
--- a/src/test/kotlin/org/wfanet/measurement/duchy/daemon/herald/HeraldTest.kt
+++ b/src/test/kotlin/org/wfanet/measurement/duchy/daemon/herald/HeraldTest.kt
@@ -124,2 +124,2 @@ internal class HeraldTest {
-  fun `syncStatuses on empty stream retains same computation token`() = runBlocking {
-    mockStreamActiveComputationsToReturn() // No items in stream.
+  fun `syncStatuses on empty stream retains same computaiton token`() = runBlocking {
+    mockStreamActiveComputationsToReturn(/* No items in stream. */ )
```

File: .github/workflows/lint.yml (hunk 1 above)
Comment (oliver-amzn) [reply .github/workflows/lint.yml:29]: 👍  thx!

File: src/main/kotlin/org/wfanet/measurement/common/RenewedFlow.kt (hunk 3 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/RenewedFlow.kt:54]: This is the case where I would be _less_ likely to convert to a block body, as the expression fits together with the last line of the signature.

File: src/main/kotlin/org/wfanet/measurement/common/crypto/SecurityProvider.kt (hunk 4 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/crypto/SecurityProvider.kt:43]: The `as X509Certificate` is necessary, as the Java security API has a single `CertificateFactory` interface despite different instances returning more specific certificate types. We want to restrict our API to `X509Certificate` as opposed to other certificate types.

File: src/main/kotlin/org/wfanet/measurement/common/identity/DuchyIds.kt (hunk 5 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/identity/DuchyIds.kt:42]: Will wrap in code fence.

File: src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt (hunk 6 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt:33]: Will do. It looks like the rule ktfmt uses for wrapping these depends on whether the expression needs parens for parameters. In this case, `Statement` is an abstract class such that the expression needs parens so it gets gets wrapped. In comparison, `TestRule` is an interface so the expression doesn't need parens.

File: src/main/kotlin/org/wfanet/measurement/common/Flow.kt (hunk 2 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/Flow.kt:142]: Will fix. Not sure why my regex missed this.

File: src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt (hunk 7 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt:87]: Will do.

File: src/main/kotlin/org/wfanet/measurement/common/Flow.kt (hunk 2 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/Flow.kt]: Done.

File: src/main/kotlin/org/wfanet/measurement/common/identity/DuchyIds.kt (hunk 5 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/identity/DuchyIds.kt:42]: Done.

File: src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt (hunk 7 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt]: Done.

File: src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt (hunk 6 above)
Comment (SanjayVas) [reply src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt]: Done.
